ts/parts-more/Polar.ts

```diff
@@ -792,23 +874,28 @@ if (seriesTypes.column) {
                             startAngleRad;
 
                 } else {
-                    point.shapeType = 'path';
                     start = barX + startAngleRad;
-                    point.shapeArgs = {
-                        d: series.polarArc(
-                            (point.yBottom as any),
-                            (point.plotY as any), start,
-                            start + point.pointWidth
-                        )
-                    };
+
+                    // Changed the way polar columns are drawn in order to make
+                    // it more consistent with the drawing of inverted columns
+                    // (they are using the same function now). Also, it was
+                    // essential to make the animation work correctly (the
+                    // scaling of the group) is replaced by animating each
+                    // element separately.
+                    point.shapeArgs = series.polarArc(
```

Comment:
`point.shapeArgs` should be an object with config for the shape, here it's SVGPathArray - why we need this inconsistency? When `point.shapeType = 'arc';` is set, then I expect `shapeArgs` be configured via `{ x, y, r, innerR, start, end }` options.

Reply:
I have changed the `shapeType` (from the `path`) when it comes to columns (only) series (no matter if inverted or not). In the actual release (v8.0.0) the radial bar series elements have `shapeType` set to `arc` while non inverted polar columns' `shapeType` are set to `path`. Because of the scaling animation problem (explained above) I have changed the way how non-inverted columns are drawn to the same way as the radial bar series elements (both have the `shapeType` set to `arc` and this way the `polarArc` function must return the `shapeArgs` object in the following form -> `{x, y, r, innerR, start, end}` for non-inverted columns too) so the function is able to return objects with arguments for arcs. Right now the `polarArc` returns such object but it also can return the path array (for columnrange series for now but this also can be changed). That's why the `SVGPathArray|SVGAttributes` is set as a possible type of returned value from the `polarArc` (Pawel's second question above). What do you think about it? Should it be done some other way?

Reply:
Can `polarArc` simply return either `{x, y, r, innerR, start, end}` (arc) or `{ d }`(path) ? And can we define different methods in proto for column and overwrite for other series (apart from columnrange - do we have other series?).

Reply:
I agree this dualism is unfortunate. I think the only sensible thing to do is that it returns `SVGAttributes` always. But we should seek consistency wherever possible. If we can use `shapeType = 'arc'` all over, it would probably be the right combination of consistency and flexibility.

Reply:
Changing the `shapeType` from `path` to `arc` for points of columnrange series allowed to get rid of the necessity of generating the path array (`SVGPathArray` type) with the `symbols.arc` (the old way, inside the `polarArc`) and instead, the `polarArc` function returns `SVGAttributes` object (only) for columnrange too (just as for column series).